pyspecProcScripts/apod_matched_filter.py

```diff
@@ -43,23 +43,26 @@ def apod_matched_filter(s, axis='t2',
     elif convolve_method == 'lorentzian':
         convolution_set = np.exp(-abs(temp.fromaxis(axis))/sigma)
     signal_E = (abs(temp * convolution_set)**2).sum(axis)
-    signal_E /= signal_E.data.max()
+    signal_E /= abs(signal_E.data).max()
     if convolve_method == 'gaussian':
         filter_width = abs(signal_E-1/sqrt(2)).argmin('sigma').item()
     elif convolve_method == 'lorentzian':
-        filter_width = abs(signal_E-signal_E.max()/2).argmin('sigma').item()
+        filter_width = abs(signal_E-1/2).argmin('sigma').item()
     logger.info(strm("FILTER WIDTH IS",filter_width))
```

Comment:
should do the same thing, but max is redundant w/ above and this way it's easier to compare to Gaussian